frontend/src/components/DomainDetails.tsx

```diff
@@ -102,25 +103,20 @@ export const DomainDetails: React.FC<Props> = (props) => {
       return [];
     }
     const ret = [];
-    if (domain.ip) {
+
```

Comment:
rm this extra empty line

Reply:
Done.
![image](https://user-images.githubusercontent.com/30377024/114319704-92223800-9ae0-11eb-8ea6-709b612eb102.png)